### PR TITLE
Ahodges/SSO teams

### PR DESCRIPTION
**Pull request by lamagnifica**
@bradenmacdonald, Beth and I talked about whether there was a need for some light-weight documentation for course teams and learners about what SSO will look like when it's available. 
This PR for https://openedx.atlassian.net/browse/DOC-2104  
- revises info in the B&R guide (for partners only) about registration
- adds info about creating an account to the (edX only) learner’s guide
- references to registering through the mobile apps deliberately left out for now (SSO not available through the apps initially)

HTML drafts available here: 
course teams: http://draft-sso-login-ca.readthedocs.org/en/latest/getting_started/accounts.html#getting-started-with-edx
students: http://draft-sso-login-students-20150721.readthedocs.org/en/latest/SFD_account.html#getting-started
@mhoeber FYI

### CODE REVIEW

File: en_us/shared/students/Section_register_account.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+#. Optionally, sign in with one of your existing accounts. 
+
+  - On edx.org, you can select your Facebook or Google account.
+
+  - On edX Edge, you can select your Facebook or Google account, or use your
+    credentials for a campus or institution account.
+
+  When you choose one of these options, some of the information that is
+  required for your new edx.org or Edge account is typically provided for you
```
Comment (mhoeber) [inline en_us/shared/students/Section_register_account.rst]: not sure what this means -- "information provided to you". provided by edX? Or third party.

**Comment by mhoeber** [comment]
@lamagnifica   this is great, thanks.  just one question

File: en_us/shared/students/Section_register_account.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/shared/students/Section_register_account.rst]: @mhoeber, yes, I struggled with how to word this. With SSO set up for OAuth2 providers, each site _can_ allow the already-known details about a user on their end to be sent back to edX so that we can pre-fill the registration form. What exactly gets sent is apparently configurable (I'm not sure exactly how yet, the specifics don't seem to be on the django admin end).  Any ideas on how I can clarify this?
(I also dislike starting a procedure with "Optionally" but couldn't think of a way around it. In the UI, I think it's pretty straightforward, so didn't want to set up a whole long weighty section about it! See http://sandbox5.opencraft.com/register)

Comment (bradenmacdonald) [inline en_us/shared/students/Section_register_account.rst]: For Google/Facebook/LinkedIn etc. (OAuth2 providers), what information is sent over for each user can be configured using the "Other Settings" field within each provider's setting in the Django admin. (For Google, you add a `SCOPE` entry to the JSON object, listing the desired properties). However, the exact way to specify these fields and the options available is different for each provider. The options are documented to some extent at http://python-social-auth.readthedocs.org/en/latest/backends/index.html#social-backends )

For SAML providers, the list of what fields are sent over is configured by the University partner (the IdP). On the Open edX end, the configuration for that IdP needs to list what fields are expected, in order to match the right information to the right field on the form (e.g. some university partners might send email over using the "email" attribute, others might send email over as the "username" or "eduUserID" field).

I would suggest rewording this paragraph along these lines:
"When you choose one of these options, some of the information that is required for your new edx.org or Edge account may be supplied by the login partner, so the registration form may be partially filled in for you. Choosing one of these options will link your new edx.org or edX Edge account to the selected partner account. Once the accounts are linked, whenever you sign in to the partner account in the future, you will be able to sign in to your edx.org or edX Edge account with a single click."

**Comment by bradenmacdonald** [comment]
@lamagnifica Your note to me in the PR description above seems to have been cut off. If you were asking me to review this PR, I just took a quick look. My only comment was on the same section as Mark.  Otherwise it looks great.

**Comment by lamagnifica** [comment]
Apologies, @bradenmacdonald not sure what happened there. Should have read something like, "Beth and I talked about whether there was a need for some light-weight documentation for course teams and learners about what SSO will look like when it's available." this is the PR for that effort!

**Comment by lamagnifica** [comment]
@JAAkana @sstack22 @ebporter for your info...
HTML drafts available here: 
course teams: http://draft-sso-login-ca.readthedocs.org/en/latest/getting_started/accounts.html#getting-started-with-edx
students: http://draft-sso-login-students-20150721.readthedocs.org/en/latest/SFD_account.html#getting-started

**Comment by lamagnifica** [comment]
@mhoeber hoping for thumbs, I'd like to get this out concurrently with the doc for partners/edge.. 
@JAAkana @sstack22 @ebporter comments always welcome

**Comment by mhoeber** [comment]
:+1:
